javascript/packages/orchestrator/src/providers/client.ts

```diff
@@ -34,7 +34,7 @@ export abstract class Client {
     providerName: string,
   ) {
     this.configPath = configPath;
-    this.namespace = namespace;
+    this.namespace = process.env.ZOMBIE_NAMESPACE || namespace;
```

Comment:
```suggestion
```
The `namespace` is initialized by the _client_ and is already cover here 
https://github.com/paritytech/zombienet/pull/2006/files#diff-8e98fb9b4dffcbe5d9f32ec1249415fdb33409effb5b6694a33aa16c5f8661a3R126-R128

Reply:
The link is broken :(

Reply:
Checked, agree, removed